### PR TITLE
Move AppVeyor tests to GitHub Actions

### PR DESCRIPTION
**Pull request by ocefpaf**
Partially addresses #1053

### CODE REVIEW

**Comment by ocefpaf** [comment]
@jswhit I'm not aware of a way to activate GitHub Actions without merging one first. I'm happy to follow up on this PR and fix it later but to activate it we would need to merge it first.

File: .github/workflows/miniconda-win.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    runs-on: windows-latest
+    strategy:
+      matrix:
+        python-version: [3.7, 3.9]
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Setup Conda
+      uses: s-weigand/setup-conda@v1
```
Comment (dopplershift) [inline .github/workflows/miniconda-win.yml:19]: Any particular reason you've opted for this one and not [conda-icubator/setup-miniconda](https://github.com/conda-incubator/setup-miniconda)?

Comment (ocefpaf) [reply .github/workflows/miniconda-win.yml:19]: It is less bloated and does less, just activate the pre-installed conda for you. Which is faster. I also had multiple issues with all the yaml options in the other one. I'd rather have the same commands on the config as those that I type on a terminal to have a 1-1 comparison if I need to debug. (With that said, I do break that rule here adding the conda-forge channel with a yaml config option :smile:)

Comment (dopplershift) [reply .github/workflows/miniconda-win.yml:19]: Makes sense.

**Comment by jswhit** [comment]
> @jswhit I'm not aware of a way to activate GitHub Actions without merging one first. I'm happy to follow up on this PR and fix it later but to activate it we would need to merge it first.

Thanks @ocefpaf - let me know when you are ready to merge and I will then merge and activate.

**Comment by ocefpaf** [comment]
If you can let's merge this now and I'll work on it tomorrow with once the status is showing on PR.

**Pull request merged**